spec/lib/os_places_api/client_spec.rb

```diff
@@ -196,6 +239,26 @@
       end
     end
 
+    context "the postcode doesn't exist in the DPA dataset" do
+      before :each do
+        Postcode.where(postcode: postcode).map(&:destroy)
+      end
+
+      it "should query OS Places API and return results from LPI dataset" do
+        os_places_api_results.delete_at(0)
+        stub_request(:get, api_endpoint)
+          .to_return(status: 200, body: successful_response.to_json)
+
+        expect(client.locations_for_postcode(postcode).as_json).to eq(
+          {
+            "average_latitude" => average_latitude,
+            "average_longitude" => average_longitude,
+            "results" => [location].as_json,
+          },
+        )
+      end
```

Comment:
I imagine if you delete line 248 this test still passes.
Could we add an explicit test that ensures that duplicate `UPRN` results are filtered out?
And also a test that both LPI and DPA results are returned if they have different UPRNs?